pygeoapi/templates/collections/tiles/index.html

```diff
@@ -39,25 +39,23 @@ <h3>Tiles</h3>
       </div>
       <br/>
       <div class="row">
-        {% if data['format']=='tilejson' %}
           <div class="col-md-2 col-sm-12">{% trans %}Metadata{% endtrans %}</div>
-          <div class="col-md-8"><a id="tilejson" href="" target="_blank">Tiles metadata in {{ data['format'] }} format</a></div>
-        {% endif %}      </div>
+          <div class="col-md-8"><a id="metadata_link" href="" target="_blank">Metadata</a></div>
+      </div>
       <script>
         var select = document.getElementById('tilingScheme');
-        var tileset = select.value;
```

Comment:
I think this is problematic, because when the page loads, we arrive [here](https://github.com/geopython/pygeoapi/pull/1482/files#diff-a33f7def52ad3a6af980fe603069ad9add9fc080d131b966551b592241dbf432R87) with an empty "tileset" variable. It could work, if the map refreshed itself, each time we choose a tileset on the dropdown.

Reply:
Fixed here https://github.com/geopython/pygeoapi/pull/1482/commits/8a7b77d514873df44ab794ce221731f8a2866710